tests/src/test/java/testdatasource/account-data.json

```diff
@@ -0,0 +1,9 @@
+{
+  "accountGuid": [
+    {
+      "validGuid": "<validGuid>",
+      "invalidGuid": "<invalidGuid>",
+      "incorrectGuid": "<IncorrectGuid>"
```

Comment:
following the values above, I believe the first char should be lower case here :)

Reply:
you can use an object like this

```java
public class AccountDataSet {

    @JsonProperty("AccountData ")
    private Map<String, String> AccountData = null;

    // getters and setters

}
```

then in the code

```java 

String json = "{ \"accountData\" : [(\"validGuid\":\"your values\" }]";
AccountDataSet accountDataSet = objectMapper.readValue(json, AccountDataSet .class); 
String validGuid = accountDataSet .getAccountData.get("validGuid");

```

Reply:
Updated. Thanks.